editors/emacs/Portfile

```diff
@@ -83,7 +83,7 @@ platform darwin {
 
 if {$subport eq $name || $subport eq "emacs-app"} {
     version         27.1
-    revision        5
+    revision        6
```

Comment:
The `revision` needs to be set back to 5. Adding a variant won't change anything for existing installs.